common/changes/dagoff-Image2_2017-01-31-23-22.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Image: Does not handle loaded event correctly in Edge for SVG files",
```

Comment:
This should read as a release note that a user can understand. For example:

Image: Now correctly loads SVG images in Edge.